src/app/shared/shared.module.ts

```diff
@@ -1,7 +1,7 @@
-import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
-import { NgxRowAccordionModule } from 'ngx-row-accordion';
+import { NgModule } from '@angular/core';
 import { RouterModule } from '@angular/router';
+import { NgxRowAccordionModule } from 'ngx-row-accordion';
```

Comment:
I need to setup the TS lint rule to auto order the imports :sweat_smile:

Reply:
Hah yea that was just my IDE running import rules